### PR TITLE
FIX: Fix simulate_evoked and apply_forward

### PR DESCRIPTION
**Pull request by larsoner**

### CODE REVIEW

File: mne/simulation/evoked.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def simulate_evoked(fwd, stc, info, cov, nave=30, iir_filter=None,
         The source time courses.
     info : dict
         Measurement info to generate the evoked.
-    cov : Covariance object
+    cov : Covariance object | None
         The noise covariance.
```
Comment (agramfort) [inline mne/simulation/evoked.py:37]: If None what is it that we do?

**Review by agramfort — APPROVED** [review]
if you can just update the docstring it would be great !

thx @larsoner !

**Comment by agramfort** [comment]
cc @LaetitiaG it should fix your pb.

**Comment by LaetitiaG** [comment]
Indeed, it works fine now! thanks a lot!

**Review by agramfort — APPROVED** [review]
@larsoner I pushed here. Merge if happy

File: mne/simulation/evoked.py [hunk 2]
```diff
@@ -70,11 +70,14 @@ def simulate_evoked(fwd, stc, info, cov, nave=30, iir_filter=None,
     .. versionadded:: 0.10.0
     """
     evoked = apply_forward(fwd, stc, info, use_cps=use_cps)
+    if cov is None:
+        return evoked
+
     if nave < np.inf:
         noise = _simulate_noise_evoked(evoked, cov, iir_filter, random_state)
         evoked.data += noise.data / math.sqrt(nave)
         evoked.nave = np.int64(nave)
```
Comment (larsoner) [inline mne/simulation/evoked.py:79]: This is incorrect as you now no longer set `nave` in the `cov=None` case

Comment (larsoner) [reply mne/simulation/evoked.py:79]: ... oh I guess this was always the case when `nave=np.inf`, but I'll fix it here

Comment (larsoner) [reply mne/simulation/evoked.py:79]: argh I see now why you did it this way. I'm okay with it!

**Pull request merged**